src/components/MetricsPane/charts/SampleEventBleachingPlot.jsx

```diff
@@ -12,31 +12,27 @@ const bleachingBenthicCategories = Object.entries(
 
 export const SampleEventBleachingPlot = ({ bleachingData }) => {
   const {
-    percent_hard_avg_avg,
-    percent_soft_avg_avg,
-    percent_algae_avg_avg,
-    quadrat_count_avg,
-    sample_unit_count,
+    percent_hard_avg_avg: percentHard,
+    percent_soft_avg_avg: percentSoft,
+    percent_algae_avg_avg: percentAlgae,
+    quadrat_count_avg: quadratCount,
+    sample_unit_count: sampleUnitCount,
```

Comment:
Lol, typing underscores is annoying. I am usually too lazy to rename python vars from the api, but I appreciate that you do it!

Is there a particular reason you do it other than preference?

Reply:
just for consistency, since I'm doing it for the other protocols code. (I'm lazy to change them too)